admin/client/App/screens/List/index.js

```diff
@@ -104,6 +105,14 @@ const ListView = React.createClass({
 				case 'sort':
 					this.props.dispatch(setActiveSort(query[key]));
 					break;
+				case 'filters':
+					try {
+						const filters = JSON.parse(query[key]);
+						setActiveFilters(filters);
+					} catch (e) {
+						console.warn('Invalid filter provided');
```

Comment:
Thinking was that the use-case was obscure enough that it did not need to be surfaced. Other example was that the existing redux action for an invalid action was to warn in the console and apply no filter.